core/src/test/java/org/apache/calcite/test/SqlFunctionsTest.java

```diff
@@ -974,4 +978,128 @@ private void thereAndBack(byte[] bytes) {
       // ok
     }
   }
+
```

Comment:
I'd rather split these tests into smaller units, basically wherever you have a comment saying "this tests xyz" I'd turn it into a separate `testNameXyz`, so that in case of test failures it's easier to understand at a glance what broke, and tests are generally easier to read since they are shorter.